### PR TITLE
Inline diagnostics with ibm cobol compiler

### PR DESCRIPTION
**Pull request by amanPras**
## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Please check the updated test cases

## Checklist:
- [x] Each of my commits contains one meaningful change
- [x] I have performed rebase of my branch on top of the development
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by ishche — APPROVED** [review]

File: server/engine/src/main/resources/resourceBundles/messages_en.properties [hunk 1]
```diff
@@ -132,6 +132,8 @@ rewriteFileOperation.notOpened=REWRITE statement can only reference a file opene
 deleteFileOperation.notOpened=DELETE statement can only reference a file opened for I-O
 startFileOperation.notOpened=START statement can only reference a file opened for INPUT or I-O
 compilerDirective.deprecatedDirectiveUse=%s is a deprecated compiler directive
+compilerDirective.warning.deprecatedDirectiveUse=The "%s" compiler option was specified, but is not supported.
```
Comment (amanPras) [inline server/engine/src/main/resources/resourceBundles/messages_en.properties]: @zeibura Please review
IBM Cobol compiler gives below warning
`IGYOS4008-W   The "NUMPROC(MIG)" compiler option was specified, but is not supported.  The option was discarded.`

File: server/engine/src/main/resources/resourceBundles/messages_en.properties [hunk 2]
```diff
@@ -132,6 +132,8 @@ rewriteFileOperation.notOpened=REWRITE statement can only reference a file opene
...
+compilerDirective.info.deprecatedDirectiveUse=The "%s" option is no longer supported.
```
Comment (amanPras) [inline server/engine/src/main/resources/resourceBundles/messages_en.properties:136]: @zeibura Please review
IBM Cobol compiler gives below information
`IGYOS4013-I   The "SIZE" option is no longer supported.`

Comment (zeibura) [reply server/engine/src/main/resources/resourceBundles/messages_en.properties:136]: this is fine

File: server/engine/src/main/resources/resourceBundles/messages_en.properties (hunk 1 above)
Comment (zeibura) [reply server/engine/src/main/resources/resourceBundles/messages_en.properties]: I would edit this to "The "NUMPROC(MIG)" compiler option is not supported. The option was ignored"

Comment (amanPras) [reply server/engine/src/main/resources/resourceBundles/messages_en.properties]: updated

**Review by zeibura — COMMENTED** [review]
see inline

**Review by zeibura — APPROVED** [review]

**Pull request merged**